doc/whats_new/v1.6.rst

```diff
@@ -310,6 +310,13 @@ Changelog
   when duplicate values in the training data lead to inaccurate outlier detection.
   :pr:`28773` by :user:`Henrique Caroço <HenriqueProj>`.
 
+:mod:`sklearn.neural_network`
+.............................
+
+- |Fix| :class:`neural_network.MLPClassifier` and :class:`neural_network.MLPRegressor`
```

Comment:
Ups, I'm the culprit here. I think that I change the changelog from the base class to the public classes. So I added MLPClassifier but then it might never have been an issue.